src/Copy/Text.elm

```diff
@@ -120,6 +120,9 @@ t key =
         Journey1Teaser ->
             "He threw my phone on the floor and smashed\u{00A0}it"
 
+        Journey1Image ->
+            "/sea-map/Person_1.svg"
```

Comment:
I guess the alts might need translating (if we have them) but the images path strings could go inline to the JourneyCard record?

Reply:
I thought they may want to shuffle order later if they were here.